serverless/src/functions/apps/handler.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyResult, APIGatewayEvent } from 'aws-lambda';
+import { formatJSONResponse } from '@libs/api-gateway';
+import * as dotenv from 'dotenv';
+import { BunnyCdn, LoadFreeCertificateMethodArgs } from '@libs/bunnyCDN';
+
+export const verifyApp = async (
+  event: APIGatewayEvent
+): Promise<APIGatewayProxyResult> => {
+  try {
+    // Check the parameters and environment variables
+    dotenv.config();
+    if (event.body === null || process.env.BUNNY_CDN_ACCESS_KEY == undefined) {
+      return formatJSONResponse({
+        status: 422,
+        message: 'Required parameters were not passed.',
+      });
+    }
+
+    // Set up constants
+    const bunnyCdn = new BunnyCdn(process.env.BUNNY_CDN_ACCESS_KEY);
+    const hostname = JSON.parse(event.body).hostname;
+
+    let args: LoadFreeCertificateMethodArgs = {
+      hostname,
+    };
+
+    await bunnyCdn.loadFreeCertificate(args).catch((e) => {
+      throw e;
+    });
```

Comment:
This code is the same than:
```suggestion
    await bunnyCdn.loadFreeCertificate(args);
```